.github/workflows/main-checks.yml

```diff
@@ -146,6 +146,38 @@ jobs:
           RUSTFLAGS: ${{ matrix.toolchain == 'nightly' && '--cfg nightly_yew' || '' }}
         run: cargo test -p yew --all-features
```

Comment:
Commenting here because GitHub won't let me select the lines above this: can we run the native/non-browser tests in WASI as well?

Reply:
> Commenting here because GitHub won't let me select the lines above this: can we run the native/non-browser tests in WASI as well?

I'll try